frontend/src/pages/EditStudent.vue

```diff
@@ -109,82 +111,68 @@ export default {
           },
           studentData: {},
           languages: [],
-          API_nativeLanguage: []
-
       }
   },
-  computed: {
-    // If NativeLanguage is changed, we assign it this value 
+  computed: {    
+    ...mapGetters(['API_nativeLanguage', 'getStudentByStudentId']),
+    ...mapState(['updateStudentSuccess']),
     nativeLanguage(){
       return this.selected ? this.selected.NativeLanguage: ''
-    }
+    },
+
```

Comment:
```suggestion
    },
    watchUpdateStudentSuccess() {
      this.myUpdateStudentSuccess = this.updateStudentSuccess
      return this.updateStudentSuccess
    },

```

local variable to follow Vuex variable value